### DISCUSSION TITLE
When will the keyboard layout fix in normal mode be functional on the new editor?

### DISCUSSION THREAD

**Discussion by VMichailidis** [esm7/obsidian-vimrc-support#97] [Q&A] [answered]
Just started with obsidian and :set langmap doesn't seem to work. What's the timeline for repairing the keyboard layout fix?

**Comment by esm7** [comment, marked as answer]
So far all my attempts to recreate this functionality in CM6 failed :-/
So the best answer I guess is "as soon as I or someone else is able to get it to work"...
